providers/base/units/suspend/suspend.pxu

```diff
@@ -251,7 +251,7 @@ category_id: com.canonical.plainbox::suspend
 id: suspend/suspend_advanced_auto
 requires:
     sleep.mem == 'supported'
-    rtc.state == 'supported'
+    rtc.state == 'supported' and rtc.wakealarm == 'supported'
```

Comment:
```suggestion
    rtc.state == 'supported'
    rtc.wakealarm == 'supported'
```

You can just place the new requirement on another line and it acts as an `and` (see [doc](https://canonical-checkbox.readthedocs-hosted.com/en/stable/reference/units/job.html#cmdoption-job-arg-requires))